ambari-web/app/styles/wizard.less

```diff
@@ -290,6 +263,19 @@
       #display-action {
         visibility:visible;
       }
+      .step3-table-progress {
+        .progress {
+          height: 8px;
+          margin: 5px 5px 5px 0;
+          border-radius: 2px;
+          width: 80%;
+        }
+      }
+      .step3-table-action {
+        .icon {
+          color: #999;
+        }
+      }
     }
```

Comment:
We really need to standardize the styles around the icons in the Actions columns of tables. That's what I introduced the ".icon-button" and associated styles. Doesn't need to be done in this pull request, but we should restyle these for consistency whenever we come across them. Overly specific style rules like this shouldn't be necessary very often.

Reply:
This was just the part of the merge with your commit I guess